dev/test-dependencies.sh

```diff
@@ -31,7 +31,7 @@ export LC_ALL=C
 # NOTE: These should match those in the release publishing script, and be kept in sync with
 #   dev/create-release/release-build.sh
 HADOOP_MODULE_PROFILES="-Phive-thriftserver -Pkubernetes -Pyarn -Phive \
-    -Pspark-ganglia-lgpl -Pkinesis-asl -Phadoop-cloud"
+    -Pspark-ganglia-lgpl -Pkinesis-asl -Phadoop-cloud -Pjvm-profiler"
```

Comment:
Like `Kafka` module, we should not include this here.

Reply:
```
$ grep kafka dev/deps/spark-deps-hadoop-3-hive-2.3 | wc -l
       0
```

Reply:
> ~Like `Kafka` module, we should not include this here.~
> 
> Do you know how we skip Kafka module's dependency here?

Let me investigate it.

Reply:
Yes, it's identical with Apache Spark's Kafka module and Apache Spark Hadoop Cloud module

- https://mvnrepository.com/artifact/org.apache.spark/spark-sql-kafka-0-10_2.13/3.5.1
- https://mvnrepository.com/artifact/org.apache.spark/spark-hadoop-cloud_2.13/3.5.1

Reply:
Yeah, let me add detailed guide in some document.

Reply:
> ~Like `Kafka` module, we should not include this here.~
> 
> Do you know how we skip Kafka module's dependency here?

I already know about it. It is implemented through `maven scope` overloading. Let me try.